Servers/Atlas/.arc/submit.py

```diff
@@ -97,8 +104,18 @@
 
 touch initial_time
 
+mkdir -p $MOLPRO_SCRDIR
+cd $MOLPRO_SCRDIR
+
+cp "$SUBMIT_DIR/input.in" .
+
 /Local/ce_dana/molpro-mpp-2022.2.3/bin/molpro -n {cpus} -t 1 input.in
```

Comment:
At the end of this command, we can add this to ensure it creates it as output.out?

`/Local/ce_dana/molpro-mpp-2022.2.3/bin/molpro -n {cpus} -t 1 input.in -o output.out`

On top of that, should we be defining `-t 1`? Rather let Molpro decide what it wants to do for the number of tasks?

Reply:
On Atlas specifically I think we should use the `-t 1` arg, we had previous conversation with David where molpro used more CPUs than allowed on atlas although it had a `-n 10` directive